src/components/Tab/Tab.js

```diff
@@ -0,0 +1,121 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { useNavigate } from 'react-router-dom';
+import {
+  BsFillFileBarGraphFill,
+  BsFillCartFill,
+  BsFillChatHeartFill,
+  BsFillBookFill,
+} from 'react-icons/bs';
+
+import { FaDumbbell } from 'react-icons/fa6';
+
+function Tab() {
+  const navigate = useNavigate();
+  const [activeTab, setActiveTab] = useState(2);
```

Comment:
초기값을 변수로 할당해서 사용해주시면 좋을 것 같습니다.
```suggestion
  const DEFAULT_TAB_ID = 2;
  ...
  const [activeTab, setActiveTab] = useState(DEFAULT_TAB_ID);
```